### DISCUSSION TITLE
Forced disable of xwayland for mainline BSPs

### DISCUSSION THREAD

**Discussion by TIS-Arne** [Freescale/meta-freescale#1889] [Q&A]
The `weston-init.bbappend` recipe always removes xwayland from the PACKAGECONFIG for mainline BSPs. The comment there says that mainline does not support xwayland. 
I am on mainline and xwayland seems to work just fine. Should this be fixed in the recipe?

**Comment by angolini** [comment]
From my perspective, I would like to see that PR
